tests/utils/dev-server.js

```diff
@@ -71,21 +71,24 @@ const startServer = async ({ cwd, offline = true, env = {}, args = [] }) => {
 
 const startDevServer = async (options, expectFailure) => {
   const maxAttempts = 5
-  for (let attempt = 1; attempt <= maxAttempts; attempt++) {
+
+  const retry = async (retriesLeft) => {
```

Comment:
Same as the other `retry` function comment

Reply:
Okay.